lib/node/node_runner/api_node_test.go

```diff
@@ -1,4 +1,4 @@
-package sebak
+package node_runner
```

Comment:
How about `noderunner` instead of `node_runner`. Golang suggest like this https://blog.golang.org/package-names .
> but do not fit well in Go:
> * computeServiceClient
> * priority_queue